browser/renderer_host/brave_render_message_filter.cc

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2019 The Brave Authors. All rights reserved.
+
+#include "brave/browser/renderer_host/brave_render_message_filter.h"
+
+#include "brave/browser/brave_browser_process_impl.h"
+#include "brave/components/brave_shields/browser/tracking_protection_service.h"
+#include "chrome/browser/content_settings/host_content_settings_map_factory.h"
+#include "chrome/browser/profiles/profile.h"
+#include "chrome/common/render_messages.h"
+
+using base::string16;
+
+BraveRenderMessageFilter::BraveRenderMessageFilter(int render_process_id,
```

Comment:
There are still lots of lines with wrong indentation. It is easy to fix just by running `depot_tools/clang-format -i` on any file you want (I'd suggest to format at least newly added files).

Reply:
Resolved!